bundles/plugins/org.bonitasoft.studio.designer.war/Retrieve UI Designer war.launch

```diff
@@ -16,5 +16,5 @@
 <listAttribute key="org.eclipse.debug.ui.favoriteGroups">
 <listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
 </listAttribute>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/org.bonitasoft.studio.pagedesigner}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/org.bonitasoft.studio.designer.war}"/>
```

Comment:
we should rename the Launch configuration n order to mention that it is the community version